test/tools/unified-spec-runner/operations.ts

```diff
@@ -258,6 +258,18 @@ operations.set('createCollection', async ({ entities, operation }) => {
 operations.set('createFindCursor', async ({ entities, operation }) => {
   const collection = entities.getEntity('collection', operation.object);
   const { filter, ...opts } = operation.arguments!;
+  switch (opts.cursorType) {
```

Comment:
Added to support spec tests in lieu of adding `cursorType` option to all cursors in this PR. Can file follow-up PR for this

Reply:
Do we want to support these options for all our cursors?  Like, what use does a tailable await data list collections cursor have?

Reply:
Fair question. I don't really see that they would have utility. Do we want to remove these options from the types for the `listCollections`, `listIndexes` and similar functions? If we do, would that constitute a breaking API change?

Reply:
Looking at the code, it seems like `listIndexes` is the only method that still incorrectly included the `tailable` and `awaitData` flags in the type for its `options` parameter, I can put up a separate PR to remove those from main.